### PR TITLE
Fixes numpydoc for util

### PR DESCRIPTION
**Pull request by kowal180**
In the process of fixing all the numpydoc problems in the util package. This is in relation to #680. 

- [x] int_to_roman

- [x] roman_to_int

- [x] calculate_luminosity

- [x] create_synpp_yaml

- [x] intensity_black_body

- [x] species_tuple_to_string

- [x] species_string_to_tuple

- [x] parse_quantity

- [x] element_symbol2atomic_number

- [x] atomic_number2element_symbol

- [x] reformat_element_symbol

- [x] quantity_linspace

- [x] convert_abundances_format

### CODE REVIEW

File: tardis/util/base.py [hunk 1]
```diff
@@ -72,6 +72,20 @@ def __str__(self):
...
+    Parameters
+    ----------
+    i: int
+        Integer to be converted into roman numerals
+
+    Returns
+    -------
+        : str
+        Returns roman numeral representation of i in str format.
+
```
Comment (wkerzendorf) [inline tardis/util/base.py]: is this newline standard for numpydoc

File: tardis/util/base.py [hunk 2]
```diff
@@ -307,6 +439,17 @@ def quantity_linspace(start, stop, num, **kwargs):
 
 
 def convert_abundances_format(fname, delimiter='\s+'):
+    """
```
Comment (wkerzendorf) [inline tardis/util/base.py]: lookup docstring indentation

File: tardis/util/base.py [hunk 3]
```diff
@@ -253,6 +354,20 @@ def parse_quantity(quantity_string):
 
 
 def element_symbol2atomic_number(element_string):
+    """
+    Takes an element symbol and returns its corresponding atomic number
+    Parameters
```
Comment (wkerzendorf) [inline tardis/util/base.py]: newline missing

File: tardis/util/base.py [hunk 4]
```diff
@@ -121,6 +151,22 @@ def calculate_luminosity(spec_fname, distance, wavelength_column=0, wavelength_u
 
 
 def create_synpp_yaml(radial1d_mdl, fname, shell_no=0, lines_db=None):
+
+    """
+
+    Parameters
+    ----------
```
Comment (wkerzendorf) [inline tardis/util/base.py]: that is not the right format

**Comment by kowal180** [comment]
- [ ] int_to_roman

- [ ] roman_to_int

- [ ] calculate_luminosity

- [ ] create_synpp_yaml

- [ ] intensity_black_body

- [ ] species_tuple_to_string

- [ ] species_string_to_tuple

- [ ] parse_quantity

- [ ] element_symbol2atomic_number

- [ ] atomic_number2element_symbol

- [ ] reformat_element_symbol

- [ ] quantity_linspace

- [ ] convert_abundances_format

File: tardis/util/base.py [hunk 5]
```diff
@@ -108,7 +122,33 @@ def roman_to_int(roman_string):
...
+    spec_fname: file or str
+        File or file name to be read
+    distance: float
+        Distance to star
+    wavelength_column: int, optional(default = 0)
+        #UNSURE
+    wavelength_unit: float, optional(default = u.angstrom)
+        Dictates units used for calculating wavelength.
+    flux_column: int, optional(default = 1)
+        #UNSURE
```
Comment (wkerzendorf) [inline tardis/util/base.py]:
```suggestion
        column index in which the flux is stored
```

File: tardis/util/base.py [hunk 6]
```diff
@@ -179,11 +229,22 @@ def create_synpp_yaml(radial1d_mdl, fname, shell_no=0, lines_db=None):
...
-        Calculate the intensity of a black-body according to the following formula
+    Calculate the intensity of a black-body according to the following formula
 
         .. math::
             I(\\nu, T) = \\frac{2h\\nu^3}{c^2}\frac{1}{e^{h\\nu \\beta_\\textrm{rad}} - 1}
 
+    Parameters
+    ----------
+    nu: int
+        Frequency of a wave of light
```
Comment (wkerzendorf) [inline tardis/util/base.py]:
```suggestion
        Frequency of light
```

File: tardis/util/base.py [hunk 7]
```diff
@@ -179,11 +229,22 @@ def create_synpp_yaml(radial1d_mdl, fname, shell_no=0, lines_db=None):
...
     """
-        Calculate the intensity of a black-body according to the following formula
+    Calculate the intensity of a black-body according to the following formula
 
         .. math::
             I(\\nu, T) = \\frac{2h\\nu^3}{c^2}\frac{1}{e^{h\\nu \\beta_\\textrm{rad}} - 1}
 
+    Parameters
+    ----------
+    nu: int
```
Comment (wkerzendorf) [inline tardis/util/base.py]: I think that is a float

File: tardis/util/base.py [hunk 8]
```diff
@@ -179,11 +229,22 @@ def create_synpp_yaml(radial1d_mdl, fname, shell_no=0, lines_db=None):
...
+        Frequency of a wave of light
+    T: int
```
Comment (wkerzendorf) [inline tardis/util/base.py]: float

File: tardis/util/base.py [hunk 9]
```diff
@@ -203,6 +280,24 @@ def species_tuple_to_string(species_tuple, roman_numerals=True):
 
 
 def species_string_to_tuple(species_string):
+    """
+    Convert a species string to its corresponding tuple representation
+
+    Parameters
+    ----------
+    species_string: str
+        String containing element symbol and number of electrons missing
```
Comment (wkerzendorf) [inline tardis/util/base.py]:
```suggestion
        String containing species symbol (e.g. Si II, Fe III)
```

File: tardis/util/base.py [hunk 10]
```diff
@@ -203,6 +280,24 @@ def species_tuple_to_string(species_tuple, roman_numerals=True):
...
+
+    Returns
+    ----------
```
Comment (wkerzendorf) [inline tardis/util/base.py]: make the minuses just as long as the Returns or Raises

File: tardis/util/base.py [hunk 11]
```diff
@@ -261,43 +387,61 @@ def element_symbol2atomic_number(element_string):
...
+    Convert atomic number to string
+
+    Parameters
+    ----------
+    atomic_number: int
+        Inputted atomic number
+
+    Returns
+    -------
+        :str
```
Comment (wkerzendorf) [inline tardis/util/base.py]: space missing

File: tardis/util/base.py [hunk 12]
```diff
@@ -253,6 +366,19 @@ def parse_quantity(quantity_string):
...
+    Takes an element symbol and returns its corresponding atomic number
+
+    Parameters
+    ----------
+    element_string: str
+        Inputted element symbol
+
+    Returns
+    -------
+        :int
```
Comment (wkerzendorf) [inline tardis/util/base.py]: space missing

**Pull request merged**